src/coreclr/src/vm/crst.h

```diff
@@ -298,6 +298,15 @@ friend class Crst;
 
     union {
         CRITICAL_SECTION    m_criticalsection;
+#if defined(TARGET_UNIX)
+	// m_criticalsection is exactly 0x90 w/ TARGET_UNIX, but smaller on Windows.
```

Comment:
I do not think that this is right: https://github.com/dotnet/runtime/blob/f44c7d1e8ddc48ab348871c25b65ca031aa0c3ba/src/coreclr/src/pal/inc/pal.h#L2348

Reply:
I just fixed the `#ifdef` last night.  I was kind of surprised the data structure was consistent across arch and build type....  It was simply because the code was being ignored.

Thanks for pointing out the PAL structure.  Having it make it easier to place a Windows version  in crosscompile.h